src/components/DiagramFrame/FramePanels/DiagramSettings.tsx

```diff
@@ -64,9 +63,9 @@ export const DiagramSettings: React.FC<DiagramSettingsProps> = ({
  }) => {
   const history = useHistory()
   return (
-    <SettingsPanel width="275px" px="medium" py="large">
+    <SettingsPanel width="275px" p="medium">
       <SpaceVertical>
-        <Heading fontSize="large">Diagram Settings</Heading>
+        <Heading fontSize="large" color={theme.colors.text4}>Diagram Settings</Heading>
```

Comment:
Is this the recommended way to define colors now?

Reply:
I would prefer a styled component but tbh I thought you liked the inline prop mechanism more. I'll follow your standard.

The prop is nice as long as it's not reused, perhaps? 

https://docs.google.com/document/d/1zyAGFHYST7oX8CHs9geyj-dOIgRVJX2Kp_zMLTetXfc/edit?usp=sharing